src/main/java/seedu/tasklist/model/assignment/UniqueAssignmentList.java

```diff
@@ -34,7 +34,7 @@
      */
     public boolean contains(Assignment toCheck) {
         requireNonNull(toCheck);
-        return internalList.stream().anyMatch(toCheck::isSameAssignment);
+        return internalList.stream().anyMatch(toCheck::equals);
```

Comment:
Is there a reason isSameAssignment causes the add command to be buggy but the equals method doesn't? Just curious XD

Reply:
cause isSameAssignment doesn't check for the module code haha, as long as name and deadline same it'll return they are same